### PR TITLE
Add Power and Volume Button support in Guest VM

### PR DESCRIPTION
**Pull request by kishanpadhiyar**
Change-Id: I0a732825d6d95b369296a5e319c849e2ed64402f
Tracked-On: OAM-95321
Signed-off-by: Zhuo Peng <zhuo.peng@intel.com>
Signed-off-by: Kishan Mochi <kishan.mochi@intel.com>

### CODE REVIEW

File: scripts/start_civ.sh [hunk 1]
```diff
@@ -558,6 +561,20 @@ function setup_qmp_pipe() {
     GUEST_QMP_PIPE="-qmp pipe:$qmp_pipe"
 }
 
+function set_guest_pwr_vol_button() {
+    echo "Guest graphic mode is $GUEST_GRAPHIC_MODE"
+    if [[ $GUEST_GRAPHIC_MODE == "GVT-d" ]]; then
```
Comment (YadongQi) [inline scripts/start_civ.sh:566]: I think no need to check graphic mode, even graphic mode is not GVT-d, we can also have this feature, right?

File: scripts/setup_host.sh [hunk 2]
```diff
@@ -197,6 +202,7 @@ function ubu_thermal_conf() {
 
 function set_host_ui() {
     if [[ $1 == "headless" ]]; then
+        setup_power_button
```
Comment (YadongQi) [inline scripts/setup_host.sh:205]: do not bind this feature to graphics mode. Add a new option in parse_arg() like "--ignore-pwr-vol-button". What do you think?

Comment (zhuopeng-sg) [reply scripts/setup_host.sh:205]: I checked with Kaushlendra, the owner of Power Management feature. He told me that they are not claiming suspend/resume feature for non-gvtd mode. So need to bind this change with gvtd check.

Comment (YadongQi) [reply scripts/setup_host.sh:205]: OK, approved

**Review by YadongQi — APPROVED** [review]

**Review by zhuopeng-sg — APPROVED** [review]

**Review by zhuopeng-sg — APPROVED** [review]

**Pull request merged**